### PR TITLE
Browser support

### PR DESCRIPTION
**Pull request by neilbulloch**
Commits with browser support, responsiveness, and final html & css files

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -5,14 +5,107 @@
...
+    <footer id="contact" class="footer_container">
+        <div class="contact_container">
+            <div class="contact_title">Contact</div>
+            <div class="mobile_number">+447474048789</div>
+            <a href="mailto:w.n.bullochjr@gmail.com" title="Link to email" class="email">w.n.bullochjr@gmail.com</a>
+        </div>
+
+        <div class="follow_container">
+                <div class="follow_btn">
+                    <a href="#"><i class="fa fa-facebook-square"></i></a>
```
Comment (sjr-mayden) [inline portfolio.html:102]: Social links don't go anywhere

File: CSS/portfolio.css [hunk 2]
```diff
@@ -0,0 +1,375 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-size: 62.5%;
+    font-family: 'Dosis', sans-serif;
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+}
+.nav_bar {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-pack: center;
+    justify-content: center;
+    padding-top: 190px;
+    font-size: 2rem;
+    width: 100%;
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+    margin-bottom: 50px;
+}
+
+.profile_pic_box {
+    width: 300px;
+    height: 300px;
+    margin-top: 30px;
+    margin-bottom: 70px;
+    text-align: center;
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+}
+
+@media screen and (max-width: 400px){
+    .profile_pic_box{
+        margin-top:10px;
+    }
+}
+
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+    border-radius: 5px 20px 5px ;
+    max-width: 100%;
+    max-height: 100%;
+}
+
+@media screen and (max-width: 600px) {
+    .profile_img {
+        max-width: 70%;
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css:148]: Unnecessary extra line

File: CSS/portfolio.css [hunk 7]
```diff
@@ -0,0 +1,375 @@
...
+    color: #F75A5C;
+}
+
+.about_me_paragraph {
+    width: 50%;
+    margin: auto;
+    font-size: 1.25rem;
+    padding-top: 10px;
+    text-align: center;
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 8]
```diff
@@ -0,0 +1,375 @@
...
+}
+
+/*section break*/
+
+.section_break{
+    border-top-style: dotted;
+    border-top-color: #F75A5C;
+    width: 70%;
+    margin: 0 auto;
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 9]
```diff
@@ -0,0 +1,375 @@
...
+}
+.projects_container {
+    width: 100%;
+
+}
+@media screen and (max-width: 600px) {
+    .projects_container{
+        padding-bottom: 10px;
+        width: 300px;
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 10]
```diff
@@ -0,0 +1,375 @@
...
+.follow_btn {
+    margin: 0;
+    padding: 0;
+    font-size: 2rem;
+    float: right;
+    width: 40px;
+}
+
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 11]
```diff
@@ -0,0 +1,375 @@
...
+    }
+}
+.follow_btn a {
+    color: floralwhite;
+    padding-bottom: 2px;
+
+}
+
+
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 12]
```diff
@@ -0,0 +1,375 @@
...
+}
+@media screen and (max-width: 600px) {
+    .follow_container {
+        padding: 30px;
+        text-align: left;
+        width: 100%;
+    }
+}
+
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css]: Unnecessary extra line

File: CSS/portfolio.css [hunk 13]
```diff
@@ -0,0 +1,375 @@
...
+    .contact_list a {
+        padding-right: 5px;
+    }
+}
+
+.contact_list a:hover {
+    color: aquamarine;
+}
+
+
```
Comment (sjr-mayden) [inline CSS/portfolio.css:357]: Unnecessary extra line

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: CSS/portfolio.css (hunk 2 above)
Comment (neilbulloch) [reply CSS/portfolio.css]: Done 👍

File: CSS/portfolio.css (hunk 3 above)
Comment (neilbulloch) [reply CSS/portfolio.css]: Done 👍

**Pull request merged**